Tests/SwiftParserTest/StringLiteralContentValueTests.swift

```diff
@@ -0,0 +1,318 @@
+  }
+
+  func testDoubleQuoted_escapedNL() {
+    test("\n")
+  }
+
+  func testDoubleQuoted_escapedCR() {
+    test("\r")
+  }
+
+  func testDoubleQuoted_escapedTab() {
+    test("\t")
+  }
+
+  func testDoubleQuoted_escapedDoubleQuote() {
+    test("\"")
+  }
+
+  func testDoubleQuoted_escapedSingleQuote() {
+    test("\'")
+  }
+
+  func testDoubleQuoted_escapedEscape() {
+    test("\\")
+  }
+
+  func testDoubleQuoted_escapedUnicodeDot() {
+    test("\u{2e}")
+  }
+
+  func testDoubleQuoted_escapedEmoji() {
+    test("\u{1F600}")
+  }
+
+  // MARK: raw double quoted string literal
+
+  func testRawDoubleQuoted_emptyString() {
+    test(#""#)
+  }
+
+  func testRawDoubleQuoted_singleSpace() {
+    test(#" "#)
+  }
+
+  func testRawDoubleQuoted_unescapedDoubleQuote() {
+    test(#"""#)
+  }
+
+  func testRawDoubleQuoted_unescapedBackslash() {
+    test(#"\"#)
+  }
+
+  func testRawDoubleQuoted_emoji() {
+    test(#"😀"#)
+  }
+
+  func testRawDoubleQuoted_escapedNul() {
+    test(#"\#0"#)
+  }
+
+  func testRawDoubleQuoted_escapedNL() {
+    test(#"\#n"#)
+  }
+
+  func testRawDoubleQuoted_escapedCR() {
+    test(#"\#r"#)
+  }
+
+  func testRawDoubleQuoted_escapedTab() {
+    test(#"\#t"#)
+  }
+
+  func testRawDoubleQuoted_escapedDoubleQuote() {
+    test(#"\#""#)
+  }
+
+  func testRawDoubleQuoted_escapedSingleQuote() {
+    test(#"\#'"#)
+  }
+
+  func testRawDoubleQuoted_escapedEscape() {
+    test(#"\#\#"#)
+  }
+
+  func testRawDoubleQuoted_escapedUnicodeDot() {
+    test(#"\#u{2e}"#)
+  }
+
+  func testRawDoubleQuoted_escapedEmoji() {
+    test(#"\#u{1F600}"#)
+  }
+
+  // MARK: multi line string literal
+
+  func testMultiLine_emptyString() {
+    test(
+      """
+
+      """
+    )
+  }
+
+  func testMultiLine_emptyLine() {
+    test(
+      """
+
+
+      """
+    )
+  }
+
+  func testMultiLine_helloWorld() {
+    test(
+      """
+      Hello, world!
+      """
+    )
+  }
+
+  func testMultiLine_indentedLines() {
+    test(
+      """
+      not indented
+        2 spaces indented
+          4 spaces indented
+      """
+    )
+  }
+
+  func testMultiLine_escapedLine() {
+    test(
+      """
+      Line 1\
+      .still on Line 1
+      """
+    )
+  }
+
+  // MARK: raw multi line string literal
+
+  func testRawMultiLine_emptyString() {
+    test(
+      #"""
+
+      """#
+    )
+  }
+
+  func testRawMultiLine_emptyLine() {
+    test(
+      #"""
+
+
+      """#
+    )
+  }
+
+  func testRawMultiLine_helloWorld() {
+    test(
+      #"""
+      Hello, world!
+      """#
+    )
+  }
+
+  func testRawMultiLine_indentedLines() {
+    test(
+      #"""
+      not indented
+        2 spaces indented
+          4 spaces indented
+      """#
+    )
+  }
+
+  func testRawMultiLine_escapedLine() {
+    test(
+      #"""
+      Line 1\
+      .still on Line 1
+      """#
+    )
+  }
+
+  // MARK: literal value not available
+
+  func testMissingQuoteStringLiteral() throws {
+    let sut = StringLiteralExprSyntax(#""a"# as ExprSyntax)!
```

Comment:
Why did you name it `sut`? `stringLiteral` would be the more natural name for me. 

Same below.

Reply:
I'm used to [name the test subject `sut`](https://en.wikipedia.org/wiki/System_under_test) in unit tests. I did not notice that it's not common in SwiftSyntax. Replaced.

Reply:
Oh, I wasn’t aware of that term. Thanks for changing it.